web/src/components/people/ExternalInvite.tsx

```diff
@@ -0,0 +1,76 @@
+import Button from "@/components/ui/Button";
+import { useState } from "react";
+import { Input } from "../shared/exports";
+import styles from "./ExternalInvite.module.css";
+import { postUsersInvite } from "@/openapi";
+import Loading from "@/components/ui/Loading";
+
+export default function ExternalInvite() {
+  const [buttonDisabled, setButtonDisabled] = useState(false);
+  const [isInputVisible, setInputVisible] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [email, setEmail] = useState("");
+  const [showSuccessMessage, setShowSuccessMessage] = useState(false);
+
+  function handleShowInput() {
+    setInputVisible(true);
+  }
+
+  async function handleSumbit(e: React.FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    try {
+      setButtonDisabled(true);
+      setIsLoading(true);
+      if (typeof email === "string") {
+        const response = await postUsersInvite({ body: { email } });
+        console.log("response", response);
+      } else {
+        // redundant maybe?
+        console.error("Email not in correct format.");
+      }
+    } catch (err) {
+      console.error("Invite post error:", err);
+    } finally {
+      setButtonDisabled(false);
+      setIsLoading(false);
+      setEmail("");
+      setShowSuccessMessage(true);
+      setTimeout(() => {
+        setShowSuccessMessage(false);
+      }, 1000);
```

Comment:
I would make the success or failure more explicit to the user. My eyes scanned around the page after I hit submit and I missed the changing of the message.